script/options.js

```diff
@@ -49,6 +52,15 @@ function applyDefaultOptions(options) {
 function applyEnvironmentOverrides(options) {
   const env = require('get-env')();
 
+  // priority order: command line option, watch task host (localhost), build type host, default host
+  if (!options.host) {
```

Comment:
@ncksllvn the watch task gets a build type argument which causes it to try to run on `http://dev.vets.gov`. Added a check for `options.watch` that will force it to use `localhost`. 

Removed the default value from `options.host` to make it easy to determine if there's a command line argument set (didn't see anything in the docs that offered this and I thought filtering the array would obfuscate this). 

Not a fan of nested ifs so I'm open to another approach.